### PR TITLE
Add working wintermesh

### PR DESCRIPTION
**Pull request by RemiDesgrange**
I bump the version of `@camptocamp/cesium`. It doesn't seems to work with a "plain" cesium.

It also apprear that building are not always shown when wintermesh is displayed. Data (b3dm files) are collected from the server but not displayed. Working on a fix.

To be noted, I had to add `ARCGIS` env var to get the composition to start.

### CODE REVIEW

File: geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ const exports = class extends ngeoOlcsManager {
   init3dTiles(visible) {
     const scene = this.ol3d.getCesiumScene();
     this.tiles3dLayers_.forEach((layer) => {
-      const url = this.tiles3dUrl_ + layer;
+      const url = this.tiles3dUrl_ + layer + "/tileset.json";
```
Comment (tonio) [inline geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js:134]: you may have checked if `tileset.json` is present before adding it.

**Review by tonio — APPROVED** [review]
Code looks, just a small comment.
Performance feels degraded to me, not sure if related to the mesh or cesium upgrade.

Let's deploy to confirm or not.

File: geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js (hunk 1 above)
Comment (RemiDesgrange) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js:134]: Like making an HTTP requests to the `tileset.json` ?

Comment (tonio) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js:134]: is present *in the url* (-:

**Pull request merged**